src/scss/components/_thumbnail.scss

```diff
@@ -9,32 +9,34 @@
   * @demo demo/gaiden-css/components/thumbnail/action/demo.html
 **/
 
-/**
-  * @styles simple Simple
-  *
-  * @demo demo/gaiden-css/components/thumbnail/simple/demo.html
-**/
-
 .thumbnail {
-  width: 100%;
-  height: 100%;
+  height: initial;
   position: relative;
   overflow: hidden;
-  border-radius: $borderRadius;
-  transition: transform .2s linear;
+  border-radius: $borderRadiusButton;
   will-change: transform;
+  margin: 0 auto;
```

Comment:
@iondrimba If we do not have a width why we are using a "margin: 0 auto". I did not understand?